google/cloud/bigtable/examples/BUILD.bazel

```diff
@@ -36,6 +36,10 @@ cc_library(
 
 load(":bigtable_examples_unit_tests.bzl", "bigtable_examples_unit_tests")
 
+EXCLUDED_UNIT_TESTS = ["howto_mock_data_api.cc"]
+
+filtered_unit_tests = [test for test in bigtable_examples_unit_tests if not test in EXCLUDED_UNIT_TESTS]
```

Comment:
This list has one item in it.

What do you think about not exporting the list to `bigtable_examples_unit_tests.bzl` at all? It would simplify the build scripts now, but note that I am bad at forecasting how these lists will grow.

Reply:
I'm a little torn between making it simpler, and making the same as the others.  It seems like something to consider though, so we can chat about it later.

Reply:
I am remembering the prior art: https://github.com/googleapis/google-cloud-cpp/blob/61ad5d2b7dff5cebefecdcf6ca3d85528b85bb0f/google/cloud/bigquery/BUILD.bazel#L85-L98

Reply:
> I'm a little torn between making it simpler, and making the same as the others.

Which "others"?  You can probably find places where we export separate lists to the same `.bzl` file, places where we use hard-coded lists in the `BUILD.bazel` and `BUILD` files, places where we use globs, and maybe other approaches that I have not thought of.

Of those, only globs look like a code smell to me.  Not that globs are *always* wrong, just that they give me pause.